components/atom/tag/src/Actionable/index.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import ActionableTagContainer from './Container'
+
+const RIGHT_ICON_PLACEMENT = 'right'
+
+const getClassNames = function ({className}) {
+  return classnames(
+    'sui-AtomTag-actionable',
+    className
+  )
+}
+
+const ActionableTag = function ({
+  Icon,
+  href,
+  iconPlacement,
+  label,
+  onClick,
+  target,
+  linkFactory,
+  className
+}) {
+  return (
+    <ActionableTagContainer className={getClassNames({className})}
+      Link={linkFactory}
+      onClick={(ev) => onClick(ev)}
+      href={href}
+      target={target}
+    >
+      {
+        Icon && iconPlacement !== RIGHT_ICON_PLACEMENT &&
+          <span className='sui-AtomTag-icon'>
+            <Icon />
+          </span>
+      }
+      <span className='sui-AtomTag-label' title={label}>
+        {label}
+      </span>
+      {
+        Icon && iconPlacement === RIGHT_ICON_PLACEMENT &&
+          <span className='sui-AtomTag-secondary-icon'>
+            <Icon />
+          </span>
+      }
+    </ActionableTagContainer>
+  )
+}
+
+ActionableTag.propTypes = {
+  className: PropTypes.string,
+  label: PropTypes.string.isRequired,
+  Icon: PropTypes.func,
```

Comment:
Isn't it a node ?

Reply:
From our last frontend meeting I understood that we'll continue using the svg components that we were using before, those are no nodes, they are funcs. For example [tag chip](https://github.com/SUI-Components/sui-components/blob/master/components/tag/chip/src/index.js#L62)

Reply:
You might be right. Don't remember and can't find the meeting note either.

Reply:
@kikoruiz @carlosvillu do you remember what we agree with?